### ISSUE TITLE
Systemd-sysusers missing, sddm trouble

### ISSUE THREAD

**Issue by poidl** [clearlinux/distribution#327]
**Is your feature request related to a problem? Please describe.**
I tried to install the desktop-kde bundle and use sddm instead of installing desktop-autostart and use gdm. One issue I encountered is that systemd-sysusers is not available to create the sddm system user. I saw that systemd-sysusers is explicitly excluded in the systemd spec file.

**Describe the solution you'd like**
Can you comment on why systemd-sysusers is not available, and what is the distribution specific way of adding system users (if any other than manual).

**Describe alternatives you've considered**
Use standard commands to create sysusers

**Additional context**
Not sure if relevant here, but I also wondered how you would configure PAM for sddm, for a “standard” desktop user experience. I ended up copying a confit file called “system-login” from the Arch Linux project, which worked fine. But I don’t know if that’s a good thing to do on clear. Thanks.

**Comment by thiagomacieira** [comment]
Clear is stateless. We don't want to create users for system services, only for real users.

So the solution is to add an sddm user for everyone. That's not a problem.

**Comment by anselmolsm** [comment]
@poidl answering your question about systemd-sysusers: it is in systemd-extras (there's an "extras" file which handles what you saw deleted in the .spec file).

Regarding PAM: It is relevant, I had to modify the pam config file shipped by sddm in order to make it work. I wonder if Clear's _login_ pam config is the replacement for _system-login_ expected by sddm.

**Comment by poidl** [comment]
Thanks for the input! I had not understood that a spec file generates multiple packages. I was looking for that systemd-extras package in the clearlinux-pkg repositories, could not find it and concluded that the package does not exist:) I looked for it because I found a reference to it in the bundle definition for one of the desktop packages. Makes sense now.

I will try modifying the sddm config as you suggest.
